src/test/java/com/znsio/teswiz/steps/AppLaunchSteps.java

```diff
@@ -67,12 +68,9 @@ private String evaluateBrowserType(String browserName) {
         String browser = details[0];
         String mobileEmulation = details[1];
 
```

Comment:
@ngourkar  - add a comment here (as documentation) to showcase from where the device name is being fetched for running in emulation mode